lms/static/sass/discussion/views/_thread.scss

```diff
@@ -25,7 +25,7 @@ body.discussion, .discussion-module {
 
     .post-header-content {
       display: inline-block;
-      width: flex-grid(9,12); 
+      width: flex-grid(8,12); 
```

Comment:
NOTE: there are cleaner ways to do this, though calc(flex-grid(9,12) - $post-image-dimension) doesn't work. :)